### DISCUSSION TITLE
[Announcement] Lecture videos for week 12

### DISCUSSION THREAD

**Discussion by KAIST-JongchanPark** [prosyslab-classroom/cs524-program-analysis#351] [Announcements]
Hi,

For those who attend FSE conference in person, we provide lecture videos for week 12.

Links are available in KLMS notice board.

Regards,
Jongchan

**Comment by greenmonn** [comment]
Hi, I think I'm reporting this too late.... but I found that the links are not about last week lectures (Advanced Iteration Techniques / Sparse Analysis). In fact, they're about abstract interpretation part which corresponds to lecture 7.

Could you check the links again and provide new ones?

**Comment by goodtaeeun** [comment]
Hi, you are right about the video links.
We updated the links to the correct ones.

Please check in the KLMS notice.

**Comment by KihongHeo** [comment]
Not too late. This error report means that you are the first.

**Comment by greenmonn** [comment]
Thank you very much!
